CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ IMPROVEMENTS
   * [\#3730](https://github.com/cosmos/cosmos-sdk/issues/3730) Improve workflow for
   `gaiad gentx` with offline public keys, by outputting stdtx file that needs to be signed.
   * [\#3761](https://github.com/cosmos/cosmos-sdk/issues/3761) Querying account related information using custom querier in auth module
+  * [\#3841](https://github.com/cosmos/cosmos-sdk/pull/3841) Add indent to JSON of `gaiacli key [add|show|list]`
```

Comment:
The `CHANGELOG` is never modified manually -- it's only updated on releases. Just take this new section entry and add it to `PENDING.md` 👍

Reply:
OK